src/tgconf_beamviewer/panels/beamviewerwidget.py

```diff
@@ -321,7 +321,7 @@ def handle_ruler_start(self):
         self._ruler_dragged = True
 
     def handle_ruler_changed(self):
-        print "handle_ruler_changed"
+        print ("handle_ruler_changed")
```

Comment:
Minor, but can't we remove the space before `(`?
```suggestion
        print("handle_ruler_changed")
```

Reply:
Done!